masterlist.yaml

```
@@ -25514,3 +25514,19 @@ plugins:
         util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 3
         udr: 3
+        
+  - name: 'SurWR.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/37982/'
+        name: 'Capital Whiterun Expansion'
+        link: 'https://www.nexusmods.com/skyrimspecialedition/mods/63355/'
```

Additional links need to be their own elements:

```suggestion
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/63355/'
```